src/app/icd/server/ICDManager.cpp

```diff
@@ -187,6 +192,13 @@ void ICDManager::SendCheckInMsgs()
                 continue;
             }
 
+            if (entry.clientType == ClientTypeEnum::kEphemeral)
+            {
+                // If the registered client is ephemeral, do not send a Check-In message
+                // continue to next entry
```

Comment:
I see no support for this behavior in the spec.  Spec says to send check-ins to all registered clients, regardless of type, no?

Reply:
A sentence was added to say that an ICD should not send Check-In messages to Ephemeral clients.
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/b3652909bdf595c7634cea1da0f1e58f837ea35f/src/data_model/ICDBehavior.adoc?plain=1#L223

Reply:
I think we should still allow to send check-in message when manually wake-up device so that controller, particularly mobile controller, has the way to know the LIT becomes active, and trigger the customized strategy if controller implements it, for example, send several stayActive command for x seconds, for example, user uses mobile controller to interact with one sensor in tough place, they may wanna interact with this lit for longer period instead of manually wake-up back and forth..

Reply:
@bzbarsky-apple @mkardous-silabs @turon

Reply:
How do you differentiate between a user using his device for normal operations vs. triggering the user active mode trigger do interact from the mobile only controller?